include/GooseEYE/Ensemble_L.hpp

```diff
@@ -0,0 +1,128 @@
+/* =================================================================================================
+
+(c - GPLv3) T.W.J. de Geus (Tom) | tom@geus.me | www.geus.me | github.com/tdegeus/GooseEYE
+
+TODO:
+- Normalization
+- Account for mask
+- Comparison sub-matrix is now copied, passing a view to the roi_L function would be faster
+
+================================================================================================= */
+
+#ifndef GOOSEEYE_ENSEMBLE_L_HPP
+#define GOOSEEYE_ENSEMBLE_L_HPP
+
+#include "GooseEYE.h"
+
+namespace GooseEYE {
+
+// -------------------------------------------------------------------------------------------------
+
+template <class T>
```

Comment:
I think it should be restricted to integral types. What do you think?

Reply:
If I'm correct, integral types does not include float or double? Aren't input image often also expressed as floating-point values?